easybuild/easyconfigs/a/annovar/annovar-2016Feb01-foss-2016a-Perl-5.22.1.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Modified by Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'Tarball'
+
+name = 'annovar'
+version = '2016Feb01'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://annovar.openbioinformatics.org/en/latest/'
+description = """ ANNOVAR is an efficient software tool to utilize update-to-date information 
+ to functionally annotate genetic variants detected from diverse genomes (including human 
+ genome hg18, hg19, hg38, as well as mouse, worm, fly, yeast and many others)."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+# Registration required for download
+# http://www.openbioinformatics.org/annovar/annovar_download_form.php
+sources = ['%(name)s.latest.tar.gz']
```

Comment:
please enforce renaming of the tarball using `%(name)s-%(version)s.tar.gz`, and include a checksum (I have `2b26e514c54a847bfdcf4ce57e43f95c` as MD5 for the version download today)